corehq/apps/fixtures/models.py

```diff
@@ -287,14 +288,37 @@ def _serialize(val):
 
     def get_groups(self, wrap=True):
         group_ids = set(
-            self.get_db().view('fixtures/ownership',
+            FixtureOwnership.get_db().view(
+                'fixtures/ownership',
                 key=[self.domain, 'group by data_item', self.get_id],
                 reduce=False,
                 wrapper=lambda r: r['value']
             )
         )
+
         if wrap:
-            return set(Group.view('_all_docs', keys=list(group_ids), include_docs=True))
+            location_group_ids = set([
```

Comment:
should add comment "if any fixtures are referencing location group IDs, make sure that those get wrapped properly as group-looking things"